tests/conftest.py

```diff
@@ -2665,35 +2665,6 @@ def cloud_data_context_with_datasource_pandas_engine(
     return context
 
 
-@pytest.fixture
-def cloud_data_context_in_cloud_mode_with_datasource_pandas_engine(
```

Comment:
Moved into data_context/conftest.py
Currently many of these fixtures are used in multiple subfolders (not just store) otherwise I would have moved them further down.